CHANGELOG.md

```diff
@@ -7,7 +7,12 @@ the release.
 
 ## Unreleased
 
-TBD
+- (Func returning) Custom sampler added to DependenciesCollector and RequestsCollector options
```

Comment:
I'd suggest to have something like:

> - OpenCensus.Collectors.AspNetCore: Allow to supply custom sampler based on request properties using custom code. For instance filter out telemetry from specific path.
> - OpenCensus.Collectors.Dependencies: Allow to supply custom sampler based on request properties using custom code. For instance, filter out calls to Zipkin REST endpoint from the exporter.

Reply:
Updated, let me know if you prefer to take off last 2 lines

Reply:
Yes, please remove last two lines. We typically list public facing things to changelog, not implementation details

Reply:
Removed